usr/lib/ica_s390_stdll/ica_specific.c

```diff
@@ -4261,8 +4261,13 @@ CK_RV token_specific_ec_generate_keypair(STDLL_TokData_t *tokdata,
     /* Generate key data for this key object */
     rc = p_ica_ec_key_generate(ica_data->adapter_handle, eckey);
     if (rc != 0) {
-        TRACE_ERROR("ica_ec_key_generate() failed with rc=%d.\n", rc);
-        ret = CKR_FUNCTION_FAILED;
+        if (rc == EINVAL) {
+            TRACE_ERROR("ica_ec_key_generate() failed with rc=%d.\n", rc);
```

Comment:
Maybe we could add `EINVAL` to the error message here?  Makes it quicker to find what is going wrong...  Similarly at other places.

Reply:
Yes, makes sense. Done.